src/utils/dig.js

```diff
@@ -61,14 +65,14 @@ export const handleDig = async ({ domain, types, short, provider }) => {
             return `${digCmd}\`\`\`\n${rowsStr}${truncStr}\n\`\`\``;
         };
 
-        // Keep adding rows until we reach Discord 4096 char limit
+        // Keep adding rows until we reach Discord 4096 char limit ()
         for (const row of sourceRows) {
-            if (output([...finalRows, row]).length > 4096) break;
+            if (output([...finalRows, row]).length > (cdflag ? 4000 : 4096)) break;
```

Comment:
```suggestion
            if (output([...finalRows, row]).length > (4096 - (cdflag ? DNSSEC_DISABLED_WARNING_MESSAGE.length : 0)) break;
```
(Might be worth moving `(4096 - (cdflag ? DNSSEC_DISABLED_WARNING_MESSAGE.length : 0)` to a max length const before this loop)